src/platform/datapath_winkernel.c

```diff
@@ -847,6 +846,11 @@ CxPlatDataPathInitialize(
         }
     }
 
+    if (!CxPlatWorkerPoolLazyStart(WorkerPool, Config)) {
```

Comment:
Launching threads here can be waste of resources? because pure winkernel doesn't use them.
Should be put in raw datapath initialization? but no need to fix in this PR I think.